.rubocop.yml

```diff
@@ -1,6 +1,26 @@
-Metrics/LineLength:
-  Max: 120
-# https://github.com/rubocop/rubocop/blob/20990ed3831589c0d9f202107b1b580ead8ef2c5/lib/rubocop/cop/style/single_line_methods.rb#L11
+inherit_gem:
+  rubocop-mdsol: rubocop.yml
+
+AllCops:
+  Exclude:
+    - "*gemspec"
+  # NOTE: Excludes vendor from linting. Needed by GitHub Actions.
+  # - https://github.com/rubocop/rubocop/issues/9832#issuecomment-873398952
+  #
+    - "vendor/bundle/**/*"
+
+Metrics/ParameterLists:
+  CountKeywordArgs: false
+
 # TODO: add "EnforcedStyle: allow_single_line" once all rubys are > 3.0 to autocorrect all single-line methods to endless
+# https://github.com/rubocop/rubocop/blob/20990ed3831589c0d9f202107b1b580ead8ef2c5/lib/rubocop/cop/style/single_line_methods.rb#L11
+#
 Style/SingleLineMethods:
   Enabled: false
+
+Style/MissingRespondToMissing:
+  Enabled: false
+
+Lint/ConstantDefinitionInBlock:
+  Exclude:
+    - "spec/**/*"
```

Comment:
We usually have these cops in our apps (usually do `double_quotes` but seems we should do `single_quotes` here). Can you add them?

```yaml
Layout/ArgumentAlignment:
  EnforcedStyle: with_fixed_indentation

Layout/FirstHashElementIndentation:
  EnforcedStyle: consistent

Style/FrozenStringLiteralComment:
  Enabled: true

Style/StringLiterals:
  Enabled: true
  EnforcedStyle: single_quotes
```

Also, please keep them in alphabetical order.